Description: Add/fix detection for sparc64
 Already applied upstream in various places
Author: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>

--- a/src/libcompiler_builtins/compiler-rt/lib/builtins/int_lib.h
+++ b/src/libcompiler_builtins/compiler-rt/lib/builtins/int_lib.h
@@ -86,7 +86,7 @@
  * Presumably it's any version of GCC, and targeting an arch that
  * does not have dedicated bit counting instructions.
  */
-#if (defined(__sparc64__) || defined(__sparcv9) || defined(__mips_n64) || defined(__mips_o64) || defined(__riscv__) \
+#if ((defined(__sparc__) && defined(__arch64__)) || defined(__mips_n64) || defined(__mips_o64) || defined(__riscv__) \
 		|| (defined(_MIPS_SIM) && ((_MIPS_SIM == _ABI64) || (_MIPS_SIM == _ABIO64))))
 si_int __clzsi2(si_int);
 si_int __ctzsi2(si_int);
